main/params.go

```diff
@@ -66,252 +67,262 @@ var (
 
 	// Maximum staking duration
-	fs.DurationVar(&Config.MaxStakeDuration, "max-stake-duration", 365*24*time.Hour, "Maximum staking duration")
+	fs.Duration(maxStakeDurationKey, 365*24*time.Hour, "Maximum staking duration")
 
 	// Stake minting period
-	fs.DurationVar(&Config.StakeMintingPeriod, "stake-minting-period", 365*24*time.Hour, "Consumption period of the staking function")
+	fs.Duration(stakeMintingPeriodKey, 365*24*time.Hour, "Consumption period of the staking function")
 
 	// Assertions:
-	fs.BoolVar(&loggingConfig.Assertions, "assertions-enabled", true, "Turn on assertion execution")
+	fs.Bool(assertionsEnabledKey, true, "Turn on assertion execution")
 
 	// Crypto:
-	fs.BoolVar(&Config.EnableCrypto, "signature-verification-enabled", true, "Turn on signature verification")
+	fs.Bool(signatureVerificationEnabledKey, true, "Turn on signature verification")
 
 	// Database:
-	db := fs.Bool("db-enabled", true, "Turn on persistent storage")
-	dbDir := fs.String("db-dir", defaultDbDir, "Database directory for Avalanche state")
+	fs.Bool(dbEnabledKey, true, "Turn on persistent storage")
+	fs.String(dbDirKey, defaultString, "Database directory for Avalanche state")
 
 	// IP:
-	consensusIP := fs.String("public-ip", "", "Public IP of this node for P2P communication. If empty, try to discover with NAT. Ignored if dynamic-public-ip is non-empty.")
+	fs.String(publicIPKey, "", "Public IP of this node for P2P communication. If empty, try to discover with NAT. Ignored if dynamic-public-ip is non-empty.")
 
 	// how often to update the dynamic IP and PnP/NAT-PMP IP and routing.
-	fs.DurationVar(&Config.DynamicUpdateDuration, "dynamic-update-duration", 5*time.Minute, "Dynamic IP and NAT Traversal update duration")
+	fs.Duration(dynamicUpdateDurationKey, 5*time.Minute, "Dynamic IP and NAT Traversal update duration")
 
-	dynamicPublicIPResolver := fs.String("dynamic-public-ip", "", "'ifconfigco' (alias 'ifconfig') or 'opendns' or 'ifconfigme'. By default does not do dynamic public IP updates. If non-empty, ignores public-ip argument.")
+	fs.String(dynamicPublicIPResolverKey, "", "'ifconfigco' (alias 'ifconfig') or 'opendns' or 'ifconfigme'. By default does not do dynamic public IP updates. If non-empty, ignores public-ip argument.")
 
 	// Incoming connection throttling
-	// After we receive [conn-meter-max-attempts] incoming connections from a given IP
+	// After we receive [conn-meter-max-conns] incoming connections from a given IP
 	// in the last [conn-meter-reset-duration], we close all subsequent incoming connections
 	// from the IP before upgrade.
-	fs.DurationVar(&Config.ConnMeterResetDuration, "conn-meter-reset-duration", 0*time.Second,
-		"Upgrade at most [conn-meter-max-attempts] connections from a given IP per [conn-meter-reset-duration]. "+
+	fs.Duration(connMeterResetDurationKey, 0*time.Second,
+		"Upgrade at most [conn-meter-max-conns] connections from a given IP per [conn-meter-reset-duration]. "+
```

Comment:
`conn-meter-max-conns` should be `connMeterResetDurationKey`

Reply:
Are you saying this should be changed in the help message? It looks correct the way it is to me.

Reply:
Yeah you're right, my bad